xml/libvirt_networking.xml

```diff
@@ -8,958 +8,976 @@
-    <title>Removing a Network</title>
-    <para>
-     To remove the definition of an inactive network from &vmhost; permanently,
-     run:
-    </para>
-<screen>&prompt.sudo;virsh net-undefine vnet_isolated
-Network vnet_isolated has been undefined</screen>
-   </sect3>
+    </step>
+    <step>
+     <para>
+      Select the interface that the VLAN device should connect to below
+      <guimenu>Real Interface for VLAN</guimenu>. If the desired interface does
+      not appear in the list, first set up this interface without an IP
+      Address.
+     </para>
+    </step>
+    <step>
+     <para>
+      Select the desired method for assigning an IP address to the VLAN device.
+     </para>
+    </step>
+    <step>
+     <para>
+      Click <guimenu>Next</guimenu> to finish the configuration.
+     </para>
+    </step>
+   </procedure>
+   <para>
+    It is also possible to use the VLAN interface as a physical interface of a
+    bridge. This makes it possible to connect several &vmhost;-only networks
+    and allows to live-migrate &vmguest; systems that are connected to such a
+    network.
+   </para>
+   <para>
+    &yast; does not always allow to set no IP address. However, this may be a
+    desired feature especially if &vmhost;-only networks should be connected.
+    In this case, use the special address <literal>0.0.0.0</literal> with
+    netmask <literal>255.255.255.255</literal>. The system scripts handle this
+    address as no IP address set.
+   </para>
   </sect2>
  </sect1>
- <sect1 xml:id="libvirt-networks-bridged">
-  <title>Bridged Networking</title>
+ <sect1 xml:id="libvirt-networks-virtual">
+  <title>Virtual Networks</title>
 
   <para>
-   A network bridge is used to connect two or more network segments. It behaves
-   like a virtual network switch, and guest machines treat it transparently as
-   a physical network interface. Any physical or virtual device can be
-   connected to the bridge.
+   &libvirt;-managed virtual networks are similar to bridged networks but
+   typically have no layer 2 connection to the &vmhost;. Connectivity to
+   the &vmhost;'s physical network is accomplished with layer 3 forwarding.
+   Virtual networks also provide DHCP and DNS services for &vmguest;s. For
+   more information on &libvirt;s virtual networks see the
+   <citetitle>Network XML format</citetitle> documentation at
+   <link xlink:href="https://libvirt.org/formatnetwork.html"/>.
   </para>
 
   <para>
-   If there is a network bridge present on &vmhost;, you can connect a
-   &vmguest; to it directly. This provides the &vmguest; with full incoming and
-   outgoing network access.
+   A standard &libvirt; installation on &productname; already comes with a
+   predefined virtual network named "default" that provides DHCP and DNS
+   services for the network along with connectivity to the &vmhost;'s physical
+   network using the network address translation (NAT) forwarding mode.
```

Comment:
I think it is best to keep "forwarding mode", since NAT is one of the modes available with the forwarding element

https://libvirt.org/formatnetwork.html#elementsConnect